.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+# Checklist:
+
+- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that validate the behaviour of the software
+- [ ] I validated that new and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] I have bumped the version of the library
```

Comment:
Does bumping the library makes sense in this project?

Reply:
Nop, removed